cmd/gadgetctl/main.go

```diff
@@ -27,6 +27,7 @@ import (
 	commonutils "github.com/inspektor-gadget/inspektor-gadget/cmd/common/utils"
 	_ "github.com/inspektor-gadget/inspektor-gadget/pkg/all-gadgets"
 	"github.com/inspektor-gadget/inspektor-gadget/pkg/environment"
+	_ "github.com/inspektor-gadget/inspektor-gadget/pkg/operators/cli"
```

Comment:
Not needed anymore now that it's being manually registered. I agree with Alban above, I think it's more clear to manually call register on the operator than relying on the init() function.